one/src/migrations.rs

```diff
@@ -51,6 +51,11 @@ pub struct FromIpfsOpts {
     #[arg(long, env = "CERAMIC_ONE_LOCAL_NETWORK_ID")]
     local_network_id: Option<u32>,
 
+    /// Expect non sharded paths to blocks.
```

Comment:
looks like some unrelated changes snuck into this PR?

Reply:
Looks like git is just confused on the pre-merge, rebased to fix it.